controllers/v1/entities.js

```diff
@@ -253,5 +253,67 @@ module.exports = class Entities extends Abstract {
       })
     }
 
+    /**
+    * @api {get} /kendra/api/v1/entities/subEntitiesRoles/:entityId
+    * Get roles based on entity type.
+    * @apiVersion 1.0.0
+    * @apiGroup user profile
```

Comment:
@aman-tunerlabs - Please change API group as per controller in all APIs for this controller.